src/sentry/eventstore/processing/service_delegator.py

```diff
@@ -0,0 +1,35 @@
+from sentry.utils.services import ServiceDelegator, make_writebehind_selector
+
+backends_config = {
+    "errors": {
+        "path": "sentry.eventstore.processing.event_processing_store",
+        "executor": {
+            "path": "sentry.utils.services.ThreadedExecutor",
+            "options": {
+                "worker_count": 1,
+            },
+        },
+    },
+    "transactions": {
+        "path": "sentry.eventstore.processing.transaction_processing_store",
+        "executor": {
+            "path": "sentry.utils.services.ThreadedExecutor",
+            "options": {
+                "worker_count": 1,
+            },
+        },
+    },
+}
+
+selector = make_writebehind_selector(
+    option_name="rc-processing-split.rollout",
+    move_to="transactions",
+    move_from="errors",
+    key_fetch=lambda *args: "a-consistent-key",
```

Comment:
This will 'work' but all operations will be hashed the same and you won't be able to incrementally move traffic.

This hook method should use the method and callargs to compute a key that varies by the inputs. An example of this can be found in getsentry/getsentry@dc0bf3605ac8c8ac8a200ac2a2dc53e93dcec851